src/sentry/issues/endpoints/group_similar_issues_embeddings.py

```diff
@@ -41,24 +41,27 @@ class GroupSimilarIssuesEmbeddingsEndpoint(GroupEndpoint):
         "GET": ApiPublishStatus.PRIVATE,
     }
 
-    def get_group_hashes_for_group_id(self, group_id: int) -> set[str]:
-        hashes = GroupHash.objects.filter(group_id=group_id)
-        return {hash.hash for hash in hashes}
+    def get_group_id_from_hashes(self, hashes: list[str], project_id: int) -> Mapping[str, int]:
```

Comment:
this function name no longer describes what this function does IMO. i would simply remove this function and combine into one statement in the caller.